2021-12-24 00:09:37,286 INFO    MainThread:1622701 [internal.py:wandb_internal():87] W&B internal server running at pid: 1622701, started at: 2021-12-24 00:09:37.285846
2021-12-24 00:09:37,288 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: check_version
2021-12-24 00:09:37,288 INFO    WriterThread:1622701 [datastore.py:open_for_write():77] open: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/run-3npg0tho.wandb
2021-12-24 00:09:37,289 DEBUG   SenderThread:1622701 [sender.py:send():238] send: header
2021-12-24 00:09:37,289 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: check_version
2021-12-24 00:09:40,305 DEBUG   SenderThread:1622701 [sender.py:send():238] send: run
2021-12-24 00:09:45,197 INFO    SenderThread:1622701 [dir_watcher.py:__init__():169] watching files in: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files
2021-12-24 00:09:45,197 INFO    SenderThread:1622701 [sender.py:_start_run_threads():825] run started: 3npg0tho with start time 1640275776
2021-12-24 00:09:45,197 DEBUG   SenderThread:1622701 [sender.py:send():238] send: summary
2021-12-24 00:09:45,198 INFO    SenderThread:1622701 [sender.py:_save_file():960] saving file wandb-summary.json with policy end
2021-12-24 00:09:45,199 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: run_start
2021-12-24 00:09:45,202 DEBUG   HandlerThread:1622701 [meta.py:__init__():40] meta init
2021-12-24 00:09:45,203 DEBUG   HandlerThread:1622701 [meta.py:__init__():54] meta init done
2021-12-24 00:09:45,203 DEBUG   HandlerThread:1622701 [meta.py:probe():214] probe
2021-12-24 00:09:45,209 DEBUG   HandlerThread:1622701 [meta.py:_setup_git():204] setup git
2021-12-24 00:09:45,223 DEBUG   HandlerThread:1622701 [meta.py:_setup_git():211] setup git done
2021-12-24 00:09:45,224 DEBUG   HandlerThread:1622701 [meta.py:_save_pip():58] save pip
2021-12-24 00:09:45,225 DEBUG   HandlerThread:1622701 [meta.py:_save_pip():72] save pip done
2021-12-24 00:09:45,225 DEBUG   HandlerThread:1622701 [meta.py:_save_conda():79] save conda
2021-12-24 00:09:46,199 INFO    Thread-8  :1622701 [dir_watcher.py:_on_file_created():217] file/dir created: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/conda-environment.yaml
2021-12-24 00:09:46,199 INFO    Thread-8  :1622701 [dir_watcher.py:_on_file_created():217] file/dir created: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/wandb-summary.json
2021-12-24 00:09:46,199 INFO    Thread-8  :1622701 [dir_watcher.py:_on_file_created():217] file/dir created: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/requirements.txt
2021-12-24 00:09:49,596 WARNING FileStreamThread:1622701 [file_stream.py:request_with_retry():461] requests_with_retry encountered retryable exception: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /files/runnphoenix/Flatland-wandb_default/3npg0tho/file_stream (Caused by ProxyError('Cannot connect to proxy.', OSError(0, 'Error'))). func: <bound method Session.post of <requests.sessions.Session object at 0x7ff798a48bb0>>, args: ('https://api.wandb.ai/files/runnphoenix/Flatland-wandb_default/3npg0tho/file_stream',), kwargs: {'json': {'files': {'wandb-summary.json': {'offset': 0, 'content': ['{}']}}, 'dropped': 0}}
2021-12-24 00:09:51,202 INFO    Thread-8  :1622701 [dir_watcher.py:_on_file_created():217] file/dir created: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/output.log
2021-12-24 00:09:53,202 INFO    Thread-8  :1622701 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/output.log
2021-12-24 00:09:55,203 INFO    Thread-8  :1622701 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/output.log
2021-12-24 00:09:58,204 INFO    Thread-8  :1622701 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/conda-environment.yaml
2021-12-24 00:09:58,379 DEBUG   HandlerThread:1622701 [meta.py:_save_conda():89] save conda done
2021-12-24 00:09:58,379 DEBUG   HandlerThread:1622701 [meta.py:probe():252] probe done
2021-12-24 00:09:58,536 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:09:58,536 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:09:59,204 INFO    Thread-8  :1622701 [dir_watcher.py:_on_file_created():217] file/dir created: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/wandb-metadata.json
2021-12-24 00:10:02,991 DEBUG   SenderThread:1622701 [sender.py:send():238] send: files
2021-12-24 00:10:02,992 INFO    SenderThread:1622701 [sender.py:_save_file():960] saving file wandb-metadata.json with policy now
2021-12-24 00:10:06,714 INFO    Thread-11 :1622701 [upload_job.py:push():137] Uploaded file /tmp/tmp3danqi7hwandb/3eh0ladn-wandb-metadata.json
2021-12-24 00:10:08,632 DEBUG   SenderThread:1622701 [sender.py:send():238] send: history
2021-12-24 00:10:08,633 DEBUG   SenderThread:1622701 [sender.py:send():238] send: summary
2021-12-24 00:10:08,636 INFO    SenderThread:1622701 [sender.py:_save_file():960] saving file wandb-summary.json with policy end
2021-12-24 00:10:09,207 INFO    Thread-8  :1622701 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/wandb-summary.json
2021-12-24 00:10:10,202 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:10:10,203 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:10:11,208 INFO    Thread-8  :1622701 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/output.log
2021-12-24 00:10:13,461 DEBUG   SenderThread:1622701 [sender.py:send():238] send: stats
2021-12-24 00:10:13,651 DEBUG   SenderThread:1622701 [sender.py:send():238] send: history
2021-12-24 00:10:13,652 DEBUG   SenderThread:1622701 [sender.py:send():238] send: summary
2021-12-24 00:10:13,654 INFO    SenderThread:1622701 [sender.py:_save_file():960] saving file wandb-summary.json with policy end
2021-12-24 00:10:14,208 INFO    Thread-8  :1622701 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/wandb-summary.json
2021-12-24 00:10:15,209 INFO    Thread-8  :1622701 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/output.log
2021-12-24 00:10:25,795 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:10:25,795 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:10:43,684 DEBUG   SenderThread:1622701 [sender.py:send():238] send: stats
2021-12-24 00:10:45,796 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:10:45,796 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:11:00,880 ERROR   SenderThread:1622701 [retry.py:__call__():126] Retry attempt failed:
Traceback (most recent call last):
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 696, in urlopen
    self._prepare_proxy(conn)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 964, in _prepare_proxy
    conn.connect()
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/connection.py", line 411, in connect
    self.sock = ssl_wrap_socket(
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 428, in ssl_wrap_socket
    ssl_sock = _ssl_wrap_socket_impl(
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 472, in _ssl_wrap_socket_impl
    return ssl_context.wrap_socket(sock, server_hostname=server_hostname)
  File "/home/runnphoenix/anaconda3/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/home/runnphoenix/anaconda3/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/home/runnphoenix/anaconda3/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
OSError: [Errno 0] Error

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 755, in urlopen
    retries = retries.increment(
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/util/retry.py", line 573, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /graphql (Caused by ProxyError('Cannot connect to proxy.', OSError(0, 'Error')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/retry.py", line 102, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 141, in execute
    return self.client.execute(*args, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/gql/transport/requests.py", line 38, in execute
    request = requests.post(self.url, **post_args)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/api.py", line 119, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/sessions.py", line 542, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/sessions.py", line 655, in send
    r = adapter.send(request, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/adapters.py", line 510, in send
    raise ProxyError(e, request=request)
requests.exceptions.ProxyError: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /graphql (Caused by ProxyError('Cannot connect to proxy.', OSError(0, 'Error')))
2021-12-24 00:11:05,797 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:11:23,604 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:11:25,797 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:11:28,102 DEBUG   SenderThread:1622701 [sender.py:send():238] send: stats
2021-12-24 00:11:28,103 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:11:33,229 INFO    Thread-8  :1622701 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/output.log
2021-12-24 00:11:45,798 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:11:51,756 ERROR   SenderThread:1622701 [retry.py:__call__():126] Retry attempt failed:
Traceback (most recent call last):
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 696, in urlopen
    self._prepare_proxy(conn)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 964, in _prepare_proxy
    conn.connect()
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/connection.py", line 411, in connect
    self.sock = ssl_wrap_socket(
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 428, in ssl_wrap_socket
    ssl_sock = _ssl_wrap_socket_impl(
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 472, in _ssl_wrap_socket_impl
    return ssl_context.wrap_socket(sock, server_hostname=server_hostname)
  File "/home/runnphoenix/anaconda3/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/home/runnphoenix/anaconda3/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/home/runnphoenix/anaconda3/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
OSError: [Errno 0] Error

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 755, in urlopen
    retries = retries.increment(
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/util/retry.py", line 573, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /graphql (Caused by ProxyError('Cannot connect to proxy.', OSError(0, 'Error')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/retry.py", line 102, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 141, in execute
    return self.client.execute(*args, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/gql/transport/requests.py", line 38, in execute
    request = requests.post(self.url, **post_args)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/api.py", line 119, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/sessions.py", line 542, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/sessions.py", line 655, in send
    r = adapter.send(request, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/adapters.py", line 510, in send
    raise ProxyError(e, request=request)
requests.exceptions.ProxyError: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /graphql (Caused by ProxyError('Cannot connect to proxy.', OSError(0, 'Error')))
2021-12-24 00:12:05,798 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:12:25,799 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:12:43,655 DEBUG   SenderThread:1622701 [sender.py:send():238] send: history
2021-12-24 00:12:43,657 DEBUG   SenderThread:1622701 [sender.py:send():238] send: summary
2021-12-24 00:12:43,661 INFO    SenderThread:1622701 [sender.py:_save_file():960] saving file wandb-summary.json with policy end
2021-12-24 00:12:43,661 DEBUG   SenderThread:1622701 [sender.py:send():238] send: stats
2021-12-24 00:12:43,661 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:12:44,248 INFO    Thread-8  :1622701 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/wandb-summary.json
2021-12-24 00:12:45,799 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:13:05,800 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:13:06,712 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:13:09,254 INFO    Thread-8  :1622701 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/output.log
2021-12-24 00:13:25,800 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:13:38,047 ERROR   SenderThread:1622701 [retry.py:__call__():126] Retry attempt failed:
Traceback (most recent call last):
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 696, in urlopen
    self._prepare_proxy(conn)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 964, in _prepare_proxy
    conn.connect()
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/connection.py", line 411, in connect
    self.sock = ssl_wrap_socket(
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 428, in ssl_wrap_socket
    ssl_sock = _ssl_wrap_socket_impl(
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 472, in _ssl_wrap_socket_impl
    return ssl_context.wrap_socket(sock, server_hostname=server_hostname)
  File "/home/runnphoenix/anaconda3/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/home/runnphoenix/anaconda3/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/home/runnphoenix/anaconda3/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
socket.timeout: _ssl.c:1106: The handshake operation timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 755, in urlopen
    retries = retries.increment(
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/util/retry.py", line 573, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /graphql (Caused by ProxyError('Cannot connect to proxy.', timeout('_ssl.c:1106: The handshake operation timed out')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/retry.py", line 102, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 141, in execute
    return self.client.execute(*args, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/gql/transport/requests.py", line 38, in execute
    request = requests.post(self.url, **post_args)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/api.py", line 119, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/sessions.py", line 542, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/sessions.py", line 655, in send
    r = adapter.send(request, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/adapters.py", line 510, in send
    raise ProxyError(e, request=request)
requests.exceptions.ProxyError: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /graphql (Caused by ProxyError('Cannot connect to proxy.', timeout('_ssl.c:1106: The handshake operation timed out')))
2021-12-24 00:13:45,800 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:14:05,801 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:14:25,801 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:14:45,803 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:15:03,284 INFO    Thread-8  :1622701 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/output.log
2021-12-24 00:15:05,804 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:15:17,060 DEBUG   SenderThread:1622701 [sender.py:send():238] send: stats
2021-12-24 00:15:17,060 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:15:20,746 DEBUG   SenderThread:1622701 [sender.py:send():238] send: stats
2021-12-24 00:15:20,747 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:15:22,283 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:15:25,047 DEBUG   SenderThread:1622701 [sender.py:send():238] send: history
2021-12-24 00:15:25,049 DEBUG   SenderThread:1622701 [sender.py:send():238] send: summary
2021-12-24 00:15:25,051 INFO    SenderThread:1622701 [sender.py:_save_file():960] saving file wandb-summary.json with policy end
2021-12-24 00:15:25,051 DEBUG   SenderThread:1622701 [sender.py:send():238] send: stats
2021-12-24 00:15:25,051 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:15:25,290 INFO    Thread-8  :1622701 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/wandb-summary.json
2021-12-24 00:15:25,804 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:15:31,190 DEBUG   SenderThread:1622701 [sender.py:send():238] send: stats
2021-12-24 00:15:31,191 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:15:32,483 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:15:36,925 DEBUG   SenderThread:1622701 [sender.py:send():238] send: stats
2021-12-24 00:15:36,925 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:15:37,949 DEBUG   SenderThread:1622701 [sender.py:send():238] send: stats
2021-12-24 00:15:37,950 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:15:38,666 DEBUG   SenderThread:1622701 [sender.py:send():238] send: history
2021-12-24 00:15:38,668 DEBUG   SenderThread:1622701 [sender.py:send():238] send: summary
2021-12-24 00:15:38,670 INFO    SenderThread:1622701 [sender.py:_save_file():960] saving file wandb-summary.json with policy end
2021-12-24 00:15:38,670 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:15:39,294 INFO    Thread-8  :1622701 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/wandb-summary.json
2021-12-24 00:15:40,919 DEBUG   SenderThread:1622701 [sender.py:send():238] send: stats
2021-12-24 00:15:40,919 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:15:45,805 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:15:45,805 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:15:46,448 DEBUG   SenderThread:1622701 [sender.py:send():238] send: stats
2021-12-24 00:16:01,449 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:16:01,450 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:16:16,216 DEBUG   SenderThread:1622701 [sender.py:send():238] send: stats
2021-12-24 00:16:18,548 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:16:18,548 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:16:35,752 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:16:35,752 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:16:46,460 DEBUG   SenderThread:1622701 [sender.py:send():238] send: stats
2021-12-24 00:16:53,057 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:16:53,057 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:17:09,235 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:17:09,235 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:17:13,182 DEBUG   SenderThread:1622701 [sender.py:send():238] send: history
2021-12-24 00:17:13,183 DEBUG   SenderThread:1622701 [sender.py:send():238] send: summary
2021-12-24 00:17:13,185 INFO    SenderThread:1622701 [sender.py:_save_file():960] saving file wandb-summary.json with policy end
2021-12-24 00:17:13,319 INFO    Thread-8  :1622701 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/wandb-summary.json
2021-12-24 00:17:15,319 INFO    Thread-8  :1622701 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/output.log
2021-12-24 00:17:16,689 DEBUG   SenderThread:1622701 [sender.py:send():238] send: stats
2021-12-24 00:17:24,788 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:17:24,789 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:17:40,569 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:17:40,569 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:17:46,917 DEBUG   SenderThread:1622701 [sender.py:send():238] send: stats
2021-12-24 00:17:57,055 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:17:57,055 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:18:13,131 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:18:13,131 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:18:17,142 DEBUG   SenderThread:1622701 [sender.py:send():238] send: stats
2021-12-24 00:18:28,694 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:18:28,694 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:18:44,789 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:18:44,790 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:18:47,371 DEBUG   SenderThread:1622701 [sender.py:send():238] send: stats
2021-12-24 00:19:00,439 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:19:00,439 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:19:17,606 DEBUG   SenderThread:1622701 [sender.py:send():238] send: stats
2021-12-24 00:19:19,792 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:19:19,792 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:19:39,792 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: stop_status
2021-12-24 00:19:47,360 INFO    Thread-8  :1622701 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/output.log
2021-12-24 00:19:49,361 INFO    Thread-8  :1622701 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/output.log
2021-12-24 00:19:53,362 INFO    Thread-8  :1622701 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/output.log
2021-12-24 00:19:53,411 ERROR   SenderThread:1622701 [retry.py:__call__():126] Retry attempt failed:
Traceback (most recent call last):
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 696, in urlopen
    self._prepare_proxy(conn)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 964, in _prepare_proxy
    conn.connect()
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/connection.py", line 411, in connect
    self.sock = ssl_wrap_socket(
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 428, in ssl_wrap_socket
    ssl_sock = _ssl_wrap_socket_impl(
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 472, in _ssl_wrap_socket_impl
    return ssl_context.wrap_socket(sock, server_hostname=server_hostname)
  File "/home/runnphoenix/anaconda3/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/home/runnphoenix/anaconda3/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/home/runnphoenix/anaconda3/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
socket.timeout: _ssl.c:1106: The handshake operation timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/connectionpool.py", line 755, in urlopen
    retries = retries.increment(
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/urllib3/util/retry.py", line 573, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /graphql (Caused by ProxyError('Cannot connect to proxy.', timeout('_ssl.c:1106: The handshake operation timed out')))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/sdk/lib/retry.py", line 102, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 141, in execute
    return self.client.execute(*args, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/wandb/vendor/gql-0.2.0/gql/transport/requests.py", line 38, in execute
    request = requests.post(self.url, **post_args)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/api.py", line 119, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/sessions.py", line 542, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/sessions.py", line 655, in send
    r = adapter.send(request, **kwargs)
  File "/home/runnphoenix/anaconda3/lib/python3.8/site-packages/requests/adapters.py", line 510, in send
    raise ProxyError(e, request=request)
requests.exceptions.ProxyError: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /graphql (Caused by ProxyError('Cannot connect to proxy.', timeout('_ssl.c:1106: The handshake operation timed out')))
2021-12-24 00:19:58,364 INFO    Thread-8  :1622701 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/output.log
2021-12-24 00:19:59,364 INFO    Thread-8  :1622701 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/output.log
2021-12-24 00:19:59,456 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:20:00,365 INFO    Thread-8  :1622701 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/output.log
2021-12-24 00:20:04,556 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:20:09,657 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:20:14,758 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:20:19,859 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:20:24,960 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:20:30,061 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:20:35,162 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:20:40,263 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:20:45,364 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:20:50,465 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:20:55,565 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:21:00,666 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:21:05,767 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:21:10,868 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:21:15,969 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:21:21,070 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:21:26,171 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:21:31,271 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:21:36,372 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:21:41,473 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:21:46,574 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:21:51,674 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:21:56,775 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:01,876 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:06,977 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:12,077 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:17,178 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:22,279 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:27,380 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:32,480 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:37,581 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:42,682 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:47,782 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:52,883 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:54,980 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: stop_status
2021-12-24 00:22:56,790 DEBUG   SenderThread:1622701 [sender.py:send():238] send: history
2021-12-24 00:22:56,792 DEBUG   SenderThread:1622701 [sender.py:send():238] send: summary
2021-12-24 00:22:56,796 INFO    SenderThread:1622701 [sender.py:_save_file():960] saving file wandb-summary.json with policy end
2021-12-24 00:22:56,796 DEBUG   SenderThread:1622701 [sender.py:send():238] send: stats
2021-12-24 00:22:56,796 DEBUG   SenderThread:1622701 [sender.py:send():238] send: history
2021-12-24 00:22:56,798 DEBUG   SenderThread:1622701 [sender.py:send():238] send: summary
2021-12-24 00:22:56,800 INFO    SenderThread:1622701 [sender.py:_save_file():960] saving file wandb-summary.json with policy end
2021-12-24 00:22:56,800 DEBUG   SenderThread:1622701 [sender.py:send():238] send: history
2021-12-24 00:22:56,802 DEBUG   SenderThread:1622701 [sender.py:send():238] send: summary
2021-12-24 00:22:56,804 INFO    SenderThread:1622701 [sender.py:_save_file():960] saving file wandb-summary.json with policy end
2021-12-24 00:22:56,804 DEBUG   SenderThread:1622701 [sender.py:send():238] send: history
2021-12-24 00:22:56,806 DEBUG   SenderThread:1622701 [sender.py:send():238] send: summary
2021-12-24 00:22:56,808 INFO    SenderThread:1622701 [sender.py:_save_file():960] saving file wandb-summary.json with policy end
2021-12-24 00:22:56,808 DEBUG   SenderThread:1622701 [sender.py:send():238] send: telemetry
2021-12-24 00:22:56,808 DEBUG   SenderThread:1622701 [sender.py:send():238] send: exit
2021-12-24 00:22:56,808 INFO    SenderThread:1622701 [sender.py:send_exit():370] handling exit code: 1
2021-12-24 00:22:56,808 INFO    SenderThread:1622701 [sender.py:send_exit():372] handling runtime: 614
2021-12-24 00:22:56,810 INFO    SenderThread:1622701 [sender.py:_save_file():960] saving file wandb-summary.json with policy end
2021-12-24 00:22:56,810 INFO    SenderThread:1622701 [sender.py:send_exit():382] send defer
2021-12-24 00:22:56,810 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,810 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,810 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,810 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,810 DEBUG   SenderThread:1622701 [sender.py:send():238] send: stats
2021-12-24 00:22:56,811 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,811 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,811 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,811 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,811 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,811 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,811 DEBUG   SenderThread:1622701 [sender.py:send():238] send: stats
2021-12-24 00:22:56,811 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,811 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,811 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,812 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,812 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,812 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,812 DEBUG   SenderThread:1622701 [sender.py:send():238] send: stats
2021-12-24 00:22:56,812 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,812 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,812 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,812 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,812 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,812 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,813 DEBUG   SenderThread:1622701 [sender.py:send():238] send: stats
2021-12-24 00:22:56,813 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,813 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,813 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: defer
2021-12-24 00:22:56,813 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,814 INFO    HandlerThread:1622701 [handler.py:handle_request_defer():147] handle defer: 0
2021-12-24 00:22:56,814 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,814 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,814 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,815 DEBUG   SenderThread:1622701 [sender.py:send():238] send: stats
2021-12-24 00:22:56,815 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,815 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,815 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,815 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,815 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,816 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,816 DEBUG   SenderThread:1622701 [sender.py:send():238] send: stats
2021-12-24 00:22:56,816 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:56,816 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: defer
2021-12-24 00:22:56,816 INFO    SenderThread:1622701 [sender.py:send_request_defer():391] handle sender defer: 0
2021-12-24 00:22:56,816 INFO    SenderThread:1622701 [sender.py:transition_state():395] send defer: 1
2021-12-24 00:22:56,816 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: defer
2021-12-24 00:22:56,816 INFO    HandlerThread:1622701 [handler.py:handle_request_defer():147] handle defer: 1
2021-12-24 00:22:56,876 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: defer
2021-12-24 00:22:56,876 INFO    SenderThread:1622701 [sender.py:send_request_defer():391] handle sender defer: 1
2021-12-24 00:22:56,876 INFO    SenderThread:1622701 [sender.py:transition_state():395] send defer: 2
2021-12-24 00:22:56,876 DEBUG   SenderThread:1622701 [sender.py:send():238] send: stats
2021-12-24 00:22:56,876 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: defer
2021-12-24 00:22:56,876 INFO    HandlerThread:1622701 [handler.py:handle_request_defer():147] handle defer: 2
2021-12-24 00:22:56,877 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: defer
2021-12-24 00:22:56,877 INFO    SenderThread:1622701 [sender.py:send_request_defer():391] handle sender defer: 2
2021-12-24 00:22:56,877 INFO    SenderThread:1622701 [sender.py:transition_state():395] send defer: 3
2021-12-24 00:22:56,877 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: defer
2021-12-24 00:22:56,877 INFO    HandlerThread:1622701 [handler.py:handle_request_defer():147] handle defer: 3
2021-12-24 00:22:56,879 DEBUG   SenderThread:1622701 [sender.py:send():238] send: summary
2021-12-24 00:22:56,881 INFO    SenderThread:1622701 [sender.py:_save_file():960] saving file wandb-summary.json with policy end
2021-12-24 00:22:56,881 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: defer
2021-12-24 00:22:56,881 INFO    SenderThread:1622701 [sender.py:send_request_defer():391] handle sender defer: 3
2021-12-24 00:22:56,881 INFO    SenderThread:1622701 [sender.py:transition_state():395] send defer: 4
2021-12-24 00:22:56,881 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: defer
2021-12-24 00:22:56,882 INFO    HandlerThread:1622701 [handler.py:handle_request_defer():147] handle defer: 4
2021-12-24 00:22:56,882 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: defer
2021-12-24 00:22:56,882 INFO    SenderThread:1622701 [sender.py:send_request_defer():391] handle sender defer: 4
2021-12-24 00:22:56,917 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:57,438 INFO    Thread-8  :1622701 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/wandb-summary.json
2021-12-24 00:22:57,551 INFO    SenderThread:1622701 [sender.py:transition_state():395] send defer: 5
2021-12-24 00:22:57,551 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:57,551 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: defer
2021-12-24 00:22:57,551 INFO    HandlerThread:1622701 [handler.py:handle_request_defer():147] handle defer: 5
2021-12-24 00:22:57,552 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: defer
2021-12-24 00:22:57,552 INFO    SenderThread:1622701 [sender.py:send_request_defer():391] handle sender defer: 5
2021-12-24 00:22:57,552 INFO    SenderThread:1622701 [dir_watcher.py:finish():283] shutting down directory watcher
2021-12-24 00:22:57,652 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:58,438 INFO    Thread-8  :1622701 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/config.yaml
2021-12-24 00:22:58,439 INFO    SenderThread:1622701 [dir_watcher.py:finish():313] scan: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files
2021-12-24 00:22:58,439 INFO    SenderThread:1622701 [dir_watcher.py:finish():327] scan save: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/config.yaml config.yaml
2021-12-24 00:22:58,439 INFO    SenderThread:1622701 [dir_watcher.py:finish():327] scan save: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/requirements.txt requirements.txt
2021-12-24 00:22:58,439 INFO    SenderThread:1622701 [dir_watcher.py:finish():327] scan save: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/output.log output.log
2021-12-24 00:22:58,439 INFO    SenderThread:1622701 [dir_watcher.py:finish():327] scan save: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/wandb-summary.json wandb-summary.json
2021-12-24 00:22:58,439 INFO    SenderThread:1622701 [dir_watcher.py:finish():327] scan save: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/conda-environment.yaml conda-environment.yaml
2021-12-24 00:22:58,440 INFO    SenderThread:1622701 [dir_watcher.py:finish():327] scan save: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/wandb-metadata.json wandb-metadata.json
2021-12-24 00:22:58,440 INFO    SenderThread:1622701 [sender.py:transition_state():395] send defer: 6
2021-12-24 00:22:58,440 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:58,441 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: defer
2021-12-24 00:22:58,444 INFO    HandlerThread:1622701 [handler.py:handle_request_defer():147] handle defer: 6
2021-12-24 00:22:58,444 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: defer
2021-12-24 00:22:58,447 INFO    SenderThread:1622701 [sender.py:send_request_defer():391] handle sender defer: 6
2021-12-24 00:22:58,451 INFO    SenderThread:1622701 [file_pusher.py:finish():177] shutting down file pusher
2021-12-24 00:22:58,542 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:58,542 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:58,643 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:58,644 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:58,745 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:58,745 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:58,847 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:58,847 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:58,948 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:58,949 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:59,050 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:59,050 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:59,152 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:59,152 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:59,253 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:59,254 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:59,355 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:59,355 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:59,457 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:59,457 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:59,558 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:59,560 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:59,661 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:59,662 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:59,757 INFO    Thread-13 :1622701 [upload_job.py:push():137] Uploaded file /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/requirements.txt
2021-12-24 00:22:59,763 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:59,763 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:59,865 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:59,865 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:59,967 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:22:59,967 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:22:59,980 INFO    Thread-16 :1622701 [upload_job.py:push():137] Uploaded file /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/conda-environment.yaml
2021-12-24 00:23:00,068 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:23:00,069 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:23:00,170 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:23:00,171 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:23:00,251 INFO    Thread-14 :1622701 [upload_job.py:push():137] Uploaded file /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/output.log
2021-12-24 00:23:00,272 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:23:00,272 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:23:00,374 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:23:00,374 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:23:00,475 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:23:00,475 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:23:00,577 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:23:00,577 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:23:00,678 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:23:00,679 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:23:00,780 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:23:00,780 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:23:00,811 INFO    Thread-12 :1622701 [upload_job.py:push():137] Uploaded file /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/config.yaml
2021-12-24 00:23:00,882 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:23:00,882 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:23:00,983 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:23:00,984 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:23:01,085 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:23:01,085 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:23:01,187 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:23:01,187 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:23:01,288 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:23:01,289 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:23:01,390 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:23:01,390 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:23:01,427 INFO    Thread-15 :1622701 [upload_job.py:push():137] Uploaded file /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/files/wandb-summary.json
2021-12-24 00:23:01,491 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:23:01,492 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:23:01,593 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:23:01,593 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:23:01,628 INFO    Thread-7  :1622701 [sender.py:transition_state():395] send defer: 7
2021-12-24 00:23:01,628 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: defer
2021-12-24 00:23:01,628 INFO    HandlerThread:1622701 [handler.py:handle_request_defer():147] handle defer: 7
2021-12-24 00:23:01,629 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: defer
2021-12-24 00:23:01,629 INFO    SenderThread:1622701 [sender.py:send_request_defer():391] handle sender defer: 7
2021-12-24 00:23:01,695 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:23:02,558 INFO    SenderThread:1622701 [sender.py:transition_state():395] send defer: 8
2021-12-24 00:23:02,558 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:23:02,559 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: defer
2021-12-24 00:23:02,559 INFO    HandlerThread:1622701 [handler.py:handle_request_defer():147] handle defer: 8
2021-12-24 00:23:02,559 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: defer
2021-12-24 00:23:02,559 INFO    SenderThread:1622701 [sender.py:send_request_defer():391] handle sender defer: 8
2021-12-24 00:23:02,559 INFO    SenderThread:1622701 [sender.py:transition_state():395] send defer: 9
2021-12-24 00:23:02,560 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: defer
2021-12-24 00:23:02,560 INFO    HandlerThread:1622701 [handler.py:handle_request_defer():147] handle defer: 9
2021-12-24 00:23:02,560 DEBUG   SenderThread:1622701 [sender.py:send():238] send: final
2021-12-24 00:23:02,560 DEBUG   SenderThread:1622701 [sender.py:send():238] send: footer
2021-12-24 00:23:02,560 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: defer
2021-12-24 00:23:02,560 INFO    SenderThread:1622701 [sender.py:send_request_defer():391] handle sender defer: 9
2021-12-24 00:23:02,660 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:23:02,660 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:23:02,661 INFO    SenderThread:1622701 [file_pusher.py:join():182] waiting for file pusher
2021-12-24 00:23:07,761 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: poll_exit
2021-12-24 00:23:09,418 DEBUG   SenderThread:1622701 [sender.py:send_request():252] send_request: poll_exit
2021-12-24 00:23:09,418 INFO    SenderThread:1622701 [file_pusher.py:join():182] waiting for file pusher
2021-12-24 00:23:09,420 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: get_summary
2021-12-24 00:23:09,423 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: sampled_history
2021-12-24 00:23:09,424 DEBUG   HandlerThread:1622701 [handler.py:handle_request():130] handle_request: shutdown
2021-12-24 00:23:09,424 INFO    HandlerThread:1622701 [handler.py:finish():731] shutting down handler
2021-12-24 00:23:09,562 INFO    WriterThread:1622701 [datastore.py:close():281] close: /home/runnphoenix/work/flatland-rl/rainbow/wandb/run-20211224_000936-3npg0tho/run-3npg0tho.wandb
2021-12-24 00:23:10,419 INFO    SenderThread:1622701 [sender.py:finish():1091] shutting down sender
2021-12-24 00:23:10,419 INFO    SenderThread:1622701 [file_pusher.py:finish():177] shutting down file pusher
2021-12-24 00:23:10,419 INFO    SenderThread:1622701 [file_pusher.py:join():182] waiting for file pusher
2021-12-24 00:23:10,420 INFO    MainThread:1622701 [internal.py:handle_exit():77] Internal process exited
